### PR TITLE
acceptanceccl: add benchmark for RESTORE of TPCH-10

### PR DESCRIPTION
**Pull request by danhhz**

### CODE REVIEW

**Review by dt — APPROVED** [review]

File: pkg/ccl/acceptanceccl/backup_test.go [hunk 1]
```diff
@@ -229,6 +230,46 @@ func BenchmarkRestoreBig(b *testing.B) {
...
+	restoreBaseURI.Path = `benchmarks/tpch/scalefactor-10`
+	restoreTPCH10URI := restoreBaseURI.String()
+
+	for _, numNodes := range []int{1, 3, 10} {
+		b.Run(fmt.Sprintf("numNodes=%d", numNodes), func(b *testing.B) {
+			if b.N != 1 {
+				b.Fatal("b.N must be 1")
+			}
+
+			const backupBaseURI = "azure://backup-test/benchmarks/tpch/scalefactor-10?AZURE_ACCOUNT_NAME=cockroachbackuptest&AZURE_ACCOUNT_KEY=FzWMkVNcHorQ2IPG5il987GEmEzSxmV8WxXOZHzKV4XfvNTXHEDdBrsAIMB2/UutSPhhNQwVt9zs4dVXD/6w/w=="
```
Comment (dt) [inline pkg/ccl/acceptanceccl/backup_test.go]: You wanna redact this key?

**Review by dt — CHANGES_REQUESTED** [review]

File: pkg/ccl/acceptanceccl/backup_test.go (hunk 1 above)
Comment (danhhz) [inline pkg/ccl/acceptanceccl/backup_test.go]: 😳

**Comment by danhhz** [comment]
RFAL

**Review by dt — APPROVED** [review]

**Comment by danhhz** [comment]
TFTR!

**Pull request merged**